File: model/src/main/java/com/sealinkin/bdef/model/Bdef_DefcartypeModel.java
package com.sealinkin.bdef.model;

import com.sealinkin.bdef.po.Bdef_Defcartype;

public class Bdef_DefcartypeModel extends Bdef_Defcartype{

	private static final long serialVersionUID = 1L;
	
	private String cartypeNo;
	private String enterpriseNo;
	public String getCartypeNo() {
		return cartypeNo;
	}
	public void setCartypeNo(String cartypeNo) {
		this.cartypeNo = cartypeNo;
	}
	public String getEnterpriseNo() {
		return enterpriseNo;
	}
	public void setEnterpriseNo(String enterpriseNo) {
		this.enterpriseNo = enterpriseNo;
	}	
}

File: wms/src/main/webapp/system/app/model/bdef/bdef_DefcartypeModel.js
/**
 * 车辆
 * @author hcx
 */
Ext.define('cms.model.bdef.bdef_DefcartypeModel', {
    extend: 'Ext.data.Model',
    idProperty:'enterpriseNo,cartypeNo',
    fields: [
        {name:'enterpriseNo'},
    	{name:'cartypeNo'},
		{name:'cartypeName'},
		{name:'cartypeWeight'},
		{name:'cartypeLength'},
		{name:'cartypeWidth'},
		{name:'cartypeHeight'},
		{name:'maxLayer'},
		{name:'rgstName'},
		{name:'rgstDate'},
		{name:'updtName'},
		{name:'updtDate'}
    ]
});